package.json

```diff
@@ -41,7 +41,7 @@
     "devDependencies": {
         "@types/fs-extra": "^8.1.0",
         "@types/mocha": "^7.0.2",
-        "cypress": "4.10.0",
+        "cypress": "5.2.0",
```

Comment:
Without updating `yarn.lock` file it will fail on Github actions as there would be versions mismatch